pyproject.toml

```diff
@@ -38,7 +24,7 @@ builtin = "clear"
 quiet-level = 3
 
 [tool.ruff]
-select = ["E", "F", "W"]
+select = ["E", "F", "W", "D201", "D204", "D206", "D207", "D208", "D209", "D210", "D211", "D214", "D215", "D300", "D301", "D403", "D405", "D406", "D407", "D408", "D409", "D410", "D411", "D412", "D414", "D418"]
```

Comment:
Is there a reason that we're not selecting `"D"`? I guess this just copies what we had before.

Reply:
No, see the previous list. We had to exclude some D rules because they require a ton of changes to our codebase.

Reply:
```python
    # Would like to enable the following rules in the future:
    # D200, D202, D205, D400
```